### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by AbdallahAmmar96**
Install travis

### CODE REVIEW

**Comment by kannanhassouna** [comment]
you should care about the database.

**Comment by AsalaKM** [comment]
Its good in this sprint
but it will changed in the next sprints

**Review by AsalaKM — COMMENTED** [review]
Its good in this sprint
but it will changed in the next sprints

**Comment by AbdallahAmmar96** [comment]
Explain !!!!
@knanahassouna1

**Review by naremanhilles — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - postgresql
+before_script:
+  - psql -c "CREATE DATABASE testing_db;" -U postgres
+  - psql -c "CREATE USER foo WITH PASSWORD 'bar';" -U postgres
+test:
+  adapter: postgresql
+  database: testing_db
+env:
+  global:
+  - PGPORT=5433
```
Comment (kannanhassouna) [inline .travis.yml]:
```suggestion
  - PGPORT=5432
```

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

**Review by kannanhassouna — APPROVED** [review]

**Review by Fatmasiam — APPROVED** [review]

**Pull request merged**